src/config.py

```diff
@@ -14,7 +14,7 @@
 
 SUMMER_SCENARIO_START = "2021-05-17"
 
-FAST_FLAG = "debug"
+FAST_FLAG = "full"
```

Comment:
Do we want to leave the default set to ``debug``?

Reply:
Yes, thanks for catching this.